shared/actions/teams.tsx

```diff
@@ -1208,6 +1209,14 @@ const getMembers = async (action: TeamsGen.GetMembersPayload, logger: Saga.SagaL
   }
 }
 
+const onSetMembers = (_: TypedState, action: TeamsGen.SetMembersPayload) => {
+  const usernameToFullname: {[username: string]: string} = {}
```

Comment:
we prefer reduce but
this likely doesn't need to be a side effect at all, the user reduce can just handle this action and do this
also note if you don't need state, you can use `chainAction`

Reply:
Other parts of the code work like this, so I figured it would be better to be consistent. Also under some definition of "side effect" this makes sense, setting the team members has the side effect of learning about their full names.

Reply:
i think the general advise of 'try and be vanilla redux' points us to not do this.  You'll notice the users reducer already handles actions from various places to update its internal state